templates/apps/sumo-aws-apps/common-org/common-org-management.yaml

```diff
@@ -952,6 +868,158 @@ Resources:
+            - !Ref AWS::Region
+          DeploymentTargets:
+              OrganizationalUnitIds:
+                - !Ref OrganizationRootID
+      TemplateURL: !Sub
+        - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}${QSVersion}/templates/apps/sumo-aws-apps/common-org/common-org-configrecorder-role.yaml'
+        - S3Region: !If [UsingDefaultBucket, !Ref 'AWS::Region', !Ref QSS3BucketRegion]
+          S3Bucket: !If [UsingDefaultBucket, !Sub '${QSS3BucketName}-${AWS::Region}', !Ref QSS3BucketName]
+
+  ConfigRecorderRoleStack:
+    Condition: create_config_recorder_role
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      Tags:
+        - Key: "version"
+          Value: !Ref QSVersion
+      TemplateURL: !Sub
+        - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}${QSVersion}/templates/apps/sumo-aws-apps/common-org/common-org-configrecorder-role.yaml'
+        - S3Region: !If [UsingDefaultBucket, !Ref 'AWS::Region', !Ref QSS3BucketRegion]
+          S3Bucket: !If [UsingDefaultBucket, !Sub '${QSS3BucketName}-${AWS::Region}', !Ref QSS3BucketName]
+      Parameters:
+        ConfigRecorderRoleName: !Sub "ConfigRecorderRole-${ParentStackName}"
+        BucketName: !If [create_s3_bucket, !GetAtt rDeliveryS3BucketArn.ARN, !Ref CloudTrailExistsS3BucketName]
+        ManagementAccountId: !Ref AWS::AccountId
+        ConfigAssumeRoleName: !Sub "Config-Recorder-${ParentStackName}"
+        OrgLambdaRoleName: !Select [1, !Split ["/", !GetAtt rSumoLogicHelperRole.Arn]]
+
+  #Get AWS Config regions
+  AWSConfigCustomResource:
+    Type: Custom::AWSConfig
+    Condition: create_config_recorder_role    
+    DependsOn:
+      - "ConfigRecorderRoleStackset"
+      - "ConfigRecorderRoleStack"
+    Properties:
+      ServiceToken: !GetAtt rSumoLogicLambdaHelper.Arn
+      ENABLED_REGIONS: !Ref ConfigRegions
+      CONFIG_ASSUME_ROLE_NAME: !Sub "Config-Recorder-${ParentStackName}"
+      CONFIG_ROLE_NAME: !Sub "ConfigRecorderRole-${ParentStackName}"
+      ALL_SUPPORTED: True
+      INCLUDE_GLOBAL_RESOURCE_TYPES: False
+      RESOURCE_TYPES: ""
+      FREQUENCY: "TwentyFour_Hours"
+      CONFIG_BUCKET: !If [create_s3_bucket, !GetAtt rDeliveryS3BucketArn.ARN, !Ref CloudTrailExistsS3BucketName] 
+
+  AWSUpdateLambdaVariablesResource:
+    Type: Custom::AWSUpdateLambdaVariables
+    Condition: create_config_recorder_role    
+    DependsOn:
+      - "ConfigRecorderRoleStackset"
+      - "ConfigRecorderRoleStack"
+    Properties:
+      ServiceToken: !GetAtt rSumoLogicLambdaHelper.Arn
+      VARIABLES: {"CONFIG_ROLE_NAME": !Sub "ConfigRecorderRole-${ParentStackName}",
+                  "ALL_SUPPORTED": "True",
+                  "INCLUDE_GLOBAL_RESOURCE_TYPES": "True",
+                  "RESOURCE_TYPES": "",
+                  "FREQUENCY": "TwentyFour_Hours",
+                  "CONFIG_BUCKET": !If [create_s3_bucket, !GetAtt rDeliveryS3BucketArn.ARN, !Ref CloudTrailExistsS3BucketName],
```

Comment:
Did we test when someone changes the config regions (add or remove config regions) Are these lambda variables getting updated during update?

Reply:
Lambda variables getting updated during update